2020-InfiniteRecharge/src/main/java/frc/robot/subsystems/PowerCell.java

```diff
@@ -0,0 +1,193 @@
+package frc.robot.subsystems;
+
+import frc.robot.OperatorInterface;
+import frc.robot.Robot;
+import frc.robot.Wiring;
+import io.github.oblarg.oblog.Loggable;
+import io.github.oblarg.oblog.annotations.Config;
+import io.github.oblarg.oblog.annotations.Log;
+
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANPIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ControlType;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
+import com.revrobotics.CANEncoder;
+
+
+ //intake has to go double the speed the shooter goes
+public class PowerCell implements Loggable{
+    //pid values
+    private final int TIMEOUT = 0;
+    private final double cLR = 0.1;
+    @Log
+    private double intake_kP = 5e-5; // P-gain = (.1*1023)/(155) = 0.66 - (350 is average error)
+    private double intake_kD = 0;
+    @Log
+    private double intake_kI = 1e-6;
+    private double shooter_kF = 0; //F-gain = (100% X 1023) / 7350 F-gain = 0.139183673 - (7350 is max speed)
+    @Log
+    private double shooter_kP = 5; // P-gain = (.1*1023)/(155) = 0.66 - (350 is average error)
+    private double shooter_kD = 0;
+    private double shooter_kI = 1e-6;
+    @Log
+    private double storage_kP = 5; // P-gain = (.1*1023)/(155) = 0.66 - (350 is average error)
```

Comment:
These comments on the gains are probably out of date.. You should consider removing them.